Makefile

```diff
@@ -5,6 +5,8 @@ setup: install-deps launch-registry
 install-deps:
 	npm install bats@1.x
 	docker pull busybox
+	command -v shellcheck || echo "  please run :	'apt install shellcheck'"
```

Comment:
Since it'd be `brew install shellcheck` on a Mac, should use https://hub.docker.com/r/koalaman/shellcheck?

Reply:
The github runner comes with shellcheck, so an image isn't needed there. To keep things consistent I have instead adjusted the readme and error message wording to be more os agnostic.